src/apitypes.ts

```diff
@@ -39,3 +41,9 @@ export interface APISearchResult {
     num_pages: number;
     per_page: number;
 }
+
+export interface APISearchOptions {
```

Comment:
This doesn't belong here, since its not strictly API because of the `language` property. Moving this to `api.ts` would be better.